From: Katharina Drexel <katharina.drexel@bfh.ch>
Date: Thu, 12 May 2022 16:25:51 +0200
Subject: Tests which do not work without vendor dir and composer environment

diff --git a/src/Symfony/Bridge/PhpUnit/Tests/DeprecationErrorHandler/DeprecationTest.php b/src/Symfony/Bridge/PhpUnit/Tests/DeprecationErrorHandler/DeprecationTest.php
index a1d3c06ea6..445b7dd9df 100644
--- a/src/Symfony/Bridge/PhpUnit/Tests/DeprecationErrorHandler/DeprecationTest.php
+++ b/src/Symfony/Bridge/PhpUnit/Tests/DeprecationErrorHandler/DeprecationTest.php
@@ -21,6 +21,9 @@ class DeprecationTest extends TestCase
     private static $vendorDir;
     private static $prefixDirsPsr4;
 
+    /**
+     * @group bridge
+     */
     private static function getVendorDir()
     {
         if (null !== self::$vendorDir) {
@@ -54,6 +57,9 @@ class DeprecationTest extends TestCase
         $this->assertSame(__FUNCTION__, $deprecation->originatingMethod());
     }
 
+    /**
+     * @group bridge
+     */
     public function testItCanTellWhetherItIsInternal()
     {
         $r = new \ReflectionClass(Deprecation::class);
@@ -169,6 +175,7 @@ class DeprecationTest extends TestCase
 
     /**
      * @dataProvider providerGetTypeDetectsSelf
+     * @group bridge
      */
     public function testGetTypeDetectsSelf(string $expectedType, string $message, string $traceClass, string $file)
     {
@@ -232,6 +239,7 @@ class DeprecationTest extends TestCase
 
     /**
      * @dataProvider providerGetTypeUsesRightTrace
+     * @group bridge
      */
     public function testGetTypeUsesRightTrace(string $expectedType, string $message, array $trace)
     {
