From: Katharina Drexel <katharina.drexel@bfh.ch>
Date: Thu, 12 May 2022 15:39:10 +0200
Subject: Amp/Loop.php throws error "Call to a member function defer() on null"

diff --git a/src/Symfony/Component/HttpClient/Tests/AmpHttpClientTest.php b/src/Symfony/Component/HttpClient/Tests/AmpHttpClientTest.php
index e17b45a0ce..d63c973396 100644
--- a/src/Symfony/Component/HttpClient/Tests/AmpHttpClientTest.php
+++ b/src/Symfony/Component/HttpClient/Tests/AmpHttpClientTest.php
@@ -14,6 +14,9 @@ namespace Symfony\Component\HttpClient\Tests;
 use Symfony\Component\HttpClient\AmpHttpClient;
 use Symfony\Contracts\HttpClient\HttpClientInterface;
 
+/**
+ * @group httpclient
+ */
 class AmpHttpClientTest extends HttpClientTestCase
 {
     protected function getHttpClient(string $testCase): HttpClientInterface
